docs/project/code_review.md

```diff
@@ -248,6 +248,19 @@ For non-optional comments, this section provides several suggestions on how best
 to make progress. If none of these work, you may need to
 [resolve an impasse or conflict](#resolving-an-impasse-or-conflict).
 
+In response to suggestions, update the files in the pull request in new commits.
+Rebasing, squashing, or force-pushing commits can break GitHub's comment
+associations, and it makes it harder to determine what's changed since the last
+review. With regular pushes, GitHub can show individual deltas, giving
+additional flexibility to the reviewer. We squash pull requests when we merge
+them, so the end result is the same.
+
+It is good to reply to every comment so that the reviewer knows you saw them.
+Best practice is to send the reply to all of the comments at once, after the
+files in the pull request have been updated. If there are a lot of replies, it
+can be helpful to include a message saying whether the pull request is now ready
+for re-review.
+
```

Comment:
Maybe also suggest flagging the reviewer to re-review using the "request review" refresh-looking button illustrated here:

![GitHub Request Review screen shot](https://user-images.githubusercontent.com/711534/189789784-fcf18d1b-137b-48ba-959a-0d05cee36c2d.png)

Reply:
Added, with some text.